(DOCSP-14159) 2.14.2 release notes (#290)
mongodb/docs-bi-connector: source/release-notes.txt

.. expect and we don't break the links on the downloads page.
.. (DOCS-9670 for context)

.. _bi-2-14-2:

|bi| 2.14.2
-----------

*Released January 27, 2021*

New Features
~~~~~~~~~~~~

> - Detached signatures are now available for |bi| releases.

Reviewer: ```suggestion
- Detached signatures are now available for release archives.
```

> Bug Fix

Reviewer: ```suggestion
Bug Fixes
```

~~~~~~~

> - Fixes an issue that caused the |bi| to generate inefficient 

Reviewer: ```suggestion
- Fixes an issue that reulted in inefficient 
```

.. expect and we don't break the links on the downloads page.
.. (DOCS-9670 for context)

.. _bi-2-14-2:

|bi| 2.14.2
-----------

*Released January 27, 2021*

> New Features
> ~~~~~~~~~~~~
> 
> - Adds the ``--configExpand`` command line option that enables the 
>   usage of expansion directives.
> 
> - Adds support for ``__rest`` and ``__exec`` in 
>   `YAML <http://yaml.org/>`_ configurations.
> 

Reviewer: ```suggestion
```
Let's leave these out of the changelog until the entire BI-2149 epic is complete

Bug Fix
~~~~~~~

> - Fixes an issue in which some queries took significantly longer to run 
>   on |bi| 2.14.

Reviewer: ```suggestion
- Fixes an issue that caused the |bi| to generate inefficient translations for ``in`` and ``or`` expressions.
```
